openerp/openupgrade/openupgrade_70.py

```diff
@@ -48,7 +46,10 @@ def set_partner_id_from_partner_address_id(
              res_partner_address as address
         WHERE address.id = target.%s""" % (table, address_field))
     for row in cr.fetchall():
-        model.write(cr, SUPERUSER_ID, row[0], {partner_field: row[1]})
+        cr.execute("UPDATE %s "
+                   "SET %s=%s "
+                   "WHERE id=%s" %
+                   (table, partner_field, row[1], row[0]))
```

Comment:
Well, it's going to be an akward string with +. As these are integer parameters, they don't need to be specially parsed. Are there injection risk?